products/redis/terraform.yaml

```diff
@@ -35,13 +35,18 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           network_name: "redis-test-network"
         test_vars_overrides:
           network_name: 'BootstrapSharedTestNetwork(t, "redis-full")'
+      - !ruby/object:Provider::Terraform::Examples
+        name: "redis_instance_private_service"
+        primary_resource_id: "cache"
     properties:
       alternativeLocationId: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
       authorizedNetwork: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
         custom_expand: 'templates/terraform/custom_expand/redis_instance_authorized_network.erb'
         diff_suppress_func: 'compareSelfLinkOrResourceName'
+      connectMode: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
For an enum like this, it may be better to set an [explicit default](https://github.com/terraform-providers/terraform-provider-google/wiki/Developer-Best-Practices#default-values) than rely on an API-side one.